### PR TITLE
Fixup TA_COUNT and TA_PERC

### LINKED ISSUE #1988: Update TA_COUNT to TA_COUNT_AK
Currently TA_COUNT is a BE signal that sends the number of tribal areas as integer to the FE. When the value is null it means there are no tribal area in the selected tract.

Currently this number populates for all tracts in AK and the CONUS.

Is there a way to turn this BE signal into TA_COUNT_AK? Meaning that it only returns the number of points in AK and does NOT count the tribal areas in the CONUS?

One way to do this would be to use the Alaskan Native Villages ID to count. This ID will only exist in AK. 

In the future we will also want a TA_COUNT_US, which will count the points in the CONUS. This is not something that is needed today, however changing TA_COUNT to TA_COUNT_AK is desired today.

### PR DESCRIPTION
**Pull request by mattbowen-usds**
Closes #1988 and #1989

### CODE REVIEW

File: data/data-pipeline/data_pipeline/etl/sources/tribal_overlap/etl.py [hunk 1]
```diff
@@ -245,12 +227,31 @@ def transform(self) -> None:
...
 
+        # Counting tribes in the lower 48 is different from counting in AK,
+        # so per request by the design and frontend team, we remove all the
+        # counts outside AK
+        merged_output_df[
+            field_names.COUNT_OF_TRIBAL_AREAS_IN_TRACT_AK
+        ] = np.where(
+            # In Alaska
+            (merged_output_df_state_fips_code == "02"),
+            # Keey the counts
```
Comment (lucasmbrown-usds) [inline data/data-pipeline/data_pipeline/etl/sources/tribal_overlap/etl.py:238]: nit: `keep`?

File: data/data-pipeline/data_pipeline/etl/sources/tribal_overlap/etl.py [hunk 2]
```diff
@@ -245,12 +227,31 @@ def transform(self) -> None:
...
+            merged_output_df[self.OVERALL_TRIBAL_COUNT],
+            # Otherwise, null them
```
Comment (lucasmbrown-usds) [inline data/data-pipeline/data_pipeline/etl/sources/tribal_overlap/etl.py:240]: great comments!

File: data/data-pipeline/data_pipeline/etl/sources/tribal_overlap/etl.py [hunk 3]
```diff
@@ -69,40 +73,18 @@ def _create_string_from_list(series: pd.Series) -> str:
...
-        rounded_percentage_str = str(round(percentage_float, 4) * 100)
-        first_digits, last_digits = rounded_percentage_str.split(".")
-
-        # Rule 3 (this is a shorthand because round(4) will truncate repeated 0s)
-        if last_digits[-1] == "0":
-            return first_digits + "%"
+            return 1.0
 
-        # Rule 4
-        if last_digits != "00":
```
Comment (lucasmbrown-usds) [inline data/data-pipeline/data_pipeline/etl/sources/tribal_overlap/etl.py:101]: We're dropping the whole rule of "Convert `7.00%` to `7`"?

**Review by lucasmbrown-usds — APPROVED** [review]
Looking good!

File: data/data-pipeline/data_pipeline/etl/sources/tribal_overlap/etl.py (hunk 3 above)
Comment (mattbowen-usds) [reply data/data-pipeline/data_pipeline/etl/sources/tribal_overlap/etl.py:101]: We're returning floats now, so it'll be on the frontend to do that

File: data/data-pipeline/data_pipeline/etl/sources/tribal_overlap/etl.py (hunk 1 above)
Comment (mattbowen-usds) [reply data/data-pipeline/data_pipeline/etl/sources/tribal_overlap/etl.py:238]: Typos will be the end of me

Comment (mattbowen-usds) [reply data/data-pipeline/data_pipeline/etl/sources/tribal_overlap/etl.py]: Fixed

**Review by esfoobar-usds — APPROVED** [review]
🍏

**Comment by vim-usds** [comment]
Checking the tiles now... let's wait until QA has approved the functionality of the new signals before merging this.

**Comment by mattbowen-usds** [comment]
@vim-usds that is great by me

**Pull request merged**